_i18n/en.yml

```diff
@@ -501,6 +501,15 @@ faq:
   avuln: >
     The Monero community has created a series of videos called "Breaking Monero", where potential Monero vulnerabilities are explored and discussed. There are 14 videos, with each exploring a different subject. Check out <a href="https://www.monerooutreach.org/breaking-monero/">the videos on monerooutreach.org</a>.
   vulnspotify: Available on Spotify as podcast
+  qcriminals: Is Monero for criminals?
+  acriminals: No. Monero is permissionless, peer-to-peer and decentralized. This means Monero developers and community have no control over how is used and who uses the technology. Monero is a tool in the hands of the public, very much like a kitchen knife or the US dollar, both of which are currently used to commit or fund horrible crimes, like terrorist attacks.
+  acriminals1: >
+    Monero, like every other currency, is money and can be used with good intentions (like funding charities) or bad intentions (like criminal activities). We cannot decide who uses Monero, but that doesn't mean we support criminal activities. Exactly like the Dollar is not considered 'for criminals', even if it's the preferred currency of terrorists, drug cartels and countless other criminal organizations.
+  qbanmonero: If you cannot avoid criminals using Monero, shouldn't it be banned?
+  abanmonero: Banning Monero would be a minor annoyance for criminals, but a major problem for those who use this currency to escape war, mass surveillance or sexual/political discrimination by tyrannical regimes or organizations.
+  abanmonero1: Banning Monero is impossible by design, but even if it was possible, the only consequence of banning it would be the sudden rise of dozens of Monero clones. Exactly how it happened with Silk Road, the pioneer darknet market. Closing it resulted in countless more darknet markets being created, many of which didn't share the ethos of the original Silk Road
+  abanmonero2: "Another example is decades of war on drugs, still ongoing in most of the world. The result of this war is in front of everyone's eyes: countless of casualties and ruined lives, but drugs have never been so many and easier to find."
+  abanmonero3: History teaches us that prohibition never succeeds, education does. If people want to do criminal activities, it's not the medium of exchange or weapon they use that makes the difference. If one is not available, they will use something else. It's the people who use it for legit reasons that will be negatively impacted.
```

Comment:
```suggestion
  abanmonero3: History teaches us that most crimes can be solved through traditional investigative methods. There is no need to resort to banning a medium of exchange that is used by many legitimate users.
```

Reply:
I believe there will be no consensus in the Monero community to the affirmation that "prohibition never succeeds". I also wouldn't touch the guns topic here, since there will be no consensus about gun prohibition.

Reply:
This is not something the community needs to be in consensus about. The consensus we care about is the historian's one.

Reply:
> I believe there will be no consensus in the Monero community to the affirmation that "prohibition never succeeds".

If that's somehow seen as a controversial statement, i'm ok with changing that phrase to something like "Prohibition is often not an efficient way to get rid of a problem. Education is"

> I also wouldn't touch the guns topic here

What do you mean? There is no gun topic here. I'm just saying that it's not the tool used to make a crime that makes the difference. We could remove it if people really want, but i would only remove that part. I'm ok with making changes if people want, but I don't agree with replacing my suggestion with yours, since it changes the paragraph completely.

Reply:
After a quick clarifying chat with selsta i agree the paragraph should be reworded, but not completely changed.